cod.md

```diff
@@ -16,18 +77,27 @@ Alternate methods of reporting issues include:
 * email [team@devfest.at](mailto:team@devfest.at)
 * call the Vienna police department at 112
 
-Anyone who is found to be in violation of this policy may be handled in any one or more of these methods, depending on the offense:
 
-* warning
-* expulsion (see below)
-* contacting of the police
+Addressing Grievances
+---------------------
 
-Actions stronger than a warning will be taken at the discretion of two or more members of the DevFest organizer team.
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify one of the event organizers with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies.
 
-False reports will be treated harshly. Personal disputes should be handled privately, unless actual harassment occurs.
 
-_DevFest Vienna reserves the right to remove any participant, sponsor or attendee at any time for any reason._
+Scope
+-----
+
+We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues—online and in-person—as well as in all one-on-one communications pertaining to community business.
+
+
+License and attribution
+-----------------------
+
+This is based on the [Berlin Code of Conduct](http://berlincodeofconduct.org/).
+
+Berlin Code of Conduct is distributed under a [Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) license. It is based on the [pdx.rb Code of Conduct](https://pdxruby.org/CONDUCT).
 
 This policy covers all organized events for the DevFest Vienna, including social events.
```

Comment:
This is the second time this line appears.

Reply:
Ahhh gotcha, thank you! I was looking for the wrong line, my bad